gluster/file/cmd/glusterfile-provisioner/glusterfile-provisioner.go

```diff
@@ -45,7 +45,7 @@ import (
 const (
 	provisionerName           = "gluster.org/glusterfile"
 	provisionerNameKey        = "PROVISIONER_NAME"
-	descAnn                   = "Gluster-external: Dynamically provisioned PV"
+	descAnn                   = "Gluster File External Provisioner"
```

Comment:
Lets discard this change. The change remove the parity between the provisioners.

Reply:
In that case I'll change `Gluster-external` to `glusterfile`, since `Gluster-external` is not a thing that exists.